CocktailsOrder/CocktailsOrder/Views/ContentView.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  ContentView.swift
+//  CocktailsOrder
+//
+//  Created by Cosma Dragos on 19.07.2022.
+//
+
+import SwiftUI
+
+struct ContentView: View {
+    var body: some View {
+        TabView {
+            MyListView()
+                .tabItem {
+                    Label("MyBar", systemImage: "list.dash")
+                }
+            
+            if let baseURL = "https://www.thecocktaildb.com", let cocktailsStore = CocktailsStore(cocktailsService: CocktailsService(baseURL: baseURL)) {
+                CocktailsList(store: cocktailsStore)
+                    .tabItem {
```

Comment:
views shouldn't be aware of any network or data layer, should use a view model instead